packages/app/src/pages/Specs/Index.vue

```diff
@@ -45,3 +45,9 @@ const title = computed(() => {
 })
 
 </script>
+
+<route>
```

Comment:
What is the intention for this update? Where is the name seen?

Reply:
![Screen Shot 2022-01-19 at 9 50 45 AM](https://user-images.githubusercontent.com/25158820/150166401-b805a157-0576-4af5-b28d-f0047372a75a.png)

This is seen on the home page.